### PR TITLE
DE5618: Program Link Not Showing

### PR DESCRIPTION
**Pull request by lukemoderwell**
**Problem**: the program link (and audio link) weren't showing even if content was added

**Solution**: add markup to show them when present

### CODE REVIEW

File: _layouts/video.html [hunk 1]
```diff
@@ -17,24 +17,38 @@
...
+          {% capture url_string %} {{ page.url | replace: '/', " " }}{% endcapture %}
+          {% assign url="" %}
+          <ol class="breadcrumb push-top hard flush-bottom">
+            <li><a class="text-cyan" href="/">Media</a></li>
+
+            {% for breadcrumb in url_array %}
+            {% capture first_word %}{{ url_string | truncatewords: 1, "" }}{% endcapture %}
+            {% capture url %}{{ url }}/{{ first_word }}{% endcapture %}
+
+            <li><a class="text-cyan" href="{{ url }}">{{ breadcrumb | replace: '-', ' ' | capitalize }}</a></li>
```
Comment (zarahzachz) [inline _layouts/video.html]: Why is `text-cyan` being added here?

File: _config.yml [hunk 2]
```diff
@@ -251,6 +251,7 @@ contentful:
           authors: author/full_name
           image: image/url
           source_url: source_url
+          audio_source_url: audio_source_url
           program: program/url
```
Comment (zarahzachz) [inline _config.yml]: What do you think about changing `programing` to `program_url` or `program_pdf`? Something that's a little more semantic.

Comment (lukemoderwell) [reply _config.yml]: I usually just match whatever is setup in the migration but that makes sense. Would we also change the migration file?

File: _layouts/video.html (hunk 1 above)
Comment (lukemoderwell) [reply _layouts/video.html]: Based on this [design](https://projects.invisionapp.com/d/#/console/13922504/291615856/preview) it looked like they were cyan.

File: _config.yml (hunk 2 above)
Comment (zarahzachz) [reply _config.yml]: No, you can change the green half of that line in the config file and leave the black half the way it is. This is just mapping the Contentful/migration side of things to our Jekyll site, so you could have it say something like `peanut_king: img/url` and wherever you reference `{{ page.peanut_king }}` you'd get the `img/url` data entered in Contentful. (It's a contrived example, but I stand by it!)

**Pull request merged**